modules/issue_tracker/help/issue_tracker.md

```diff
@@ -0,0 +1,15 @@
+## Issue Tracker Help
```

Comment:
The markdown help only works for "new" style modules which have 1 class per page, so this won't show up in the front end since the issue_tracker hasn't been converted.

(That said, I'm not opposed to merging this anyways if someone's reviewed the content..)